content/post/mongodb-arbiter-2022.06.01/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Managing MongoDB with Arbiter on Kubernetes using KubeDB
+date: 2022-06-01
+weight: 20
+authors:
+- Arnob kumar saha
+tags:
+- kubedb
+- MongoDB
+- Arbiter
+- kubernetes
+- replicaset
+- shard
+- cert-manager
+- cost reducing
+- opsRequest
+- verticalScaling
+- Reconfigure
+- enterprise
+- community
+---
+
+## Summary
+
+On 1st June 2022, Appscode held a webinar on **Managing MongoDB with Arbiter on Kubernetes using KubeDB**. Key contents of the webinars are:
+- Introducing the concept of MongoDB Arbiter
+- Importance of voting in MongoDB
+- How Arbiter works on Replica-set database & Sharded cluster.
+- Live Demonstration
+- Q & A Session.
+
+
+
+## Description of the Webinar
+
+Earlier in this webinar, The speaker discussed about the `Arbiter` feature for `MongoDB`. Where He showed how to use kubeDB’s newly introduced features MongoDB-Arbiter. 
+He discussed the core concept behind replicaSet & shared-cluster architectures. He also talked about the primary-Election mechanism for automatic failover.
+
+
+Later in this webinar, He created an arbiter-enabled replicaset database & a verticalScale opsRequest to show how to scale the resources for each pod & to show how the election makes impact.
+He also applied an arbiter-enabled sharded cluster & Reconfigure opsRequest to show how to change the db-specific internal configurations on the fly.
```

Comment:
We also applied an arbiter-enabled sharded cluster & Reconfigure opsRequest to show how to change the db-specific internal configurations on the fly.